Makefile

```diff
@@ -1,9 +1,26 @@
 CACHITO_COMPOSE_ENGINE ?= docker-compose
 PYTHON_VERSION_VENV ?= python3.8
 TOX_ENVLIST ?= py38
-TOX_ARGS ?= 
-ifeq ($(CACHITO_COMPOSE_ENGINE), docker-compose)
-	DOWN_OPTS=-v
+TOX_ARGS ?=
+
+PODMAN_COMPOSE_AUTO_URL ?= https://raw.githubusercontent.com/containers/podman-compose/devel/podman_compose.py
+PODMAN_COMPOSE_TMP ?= tmp/podman_compose.py
+
+ifeq (podman-compose-auto,$(CACHITO_COMPOSE_ENGINE))
+ifeq (,$(wildcard $(PODMAN_COMPOSE_TMP)))
+$(shell mkdir -p `dirname $(PODMAN_COMPOSE_TMP)`)
+$(shell curl -sL $(PODMAN_COMPOSE_AUTO_URL) -o $(PODMAN_COMPOSE_TMP))
+$(shell chmod +x $(PODMAN_COMPOSE_TMP))
```

Comment:
Since podman-compose requires `pyyaml`, it might be a good idea to to install it into a venv as @arewm suggested. Or at least document that you need pyyaml installed for this to work.

Reply:
I took the coward approach and just added a documentation item. At this point, I think creating a venv for podman-compose is overkill.